packages/venia-ui/lib/components/App/app.js

```diff
@@ -61,15 +61,10 @@ const App = props => {
                 timeout: 15000,
                 type: 'error'
             };
-            // Only add a toast for new errors. Without this condition we would
-            // re-add toasts when one error is removed even if there were two
-            // added at the same time.
-            const errorToastId = getToastId(errorToastProps);
```

Comment:
I wrote this function specifically because of this bug. Now that the bug is no longer present we don't really have a need to export it aside from it just being part of the component API.